src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -444,9 +437,11 @@ static bool IsComInterop (IMarshalInfoProvider marshalInfoProvider, TypeReferenc
 		private static bool ComDangerousMethod (MethodDefinition methodDefinition, LinkContext context)
 		{
 			bool comDangerousMethod = IsComInterop (methodDefinition.MethodReturnType, methodDefinition.ReturnType, context);
+#pragma warning disable RS0030
```

Comment:
Please add a comment what the warning RS0030 is... and why it's OK to violate it here